loader/src/test/java/loader/TripleTableLoaderDefaultPartTest.java

```diff
@@ -0,0 +1,440 @@
+		t2.setS("<http://example.org/resource/Res1>");
+		t2.setP("<http://example.org/property/pro1>");
+		t2.setO("<http://example.org/resource/Res3>");
+
+		TripleBean t3 = new TripleBean();
+		t3.setS("<http://example.org/resource/Res5>");
+		t3.setP("<http://example.org/property/pro1>");
+		t3.setO("<http://example.org/resource/Res1>");
+
+		// These are duplicate triples
+		TripleBean t4 = new TripleBean();
+		t4.setS("<http://example.org/resource/Res1>");
+		t4.setP("<http://example.org/property/pro1>");
+		t4.setO("<http://example.org/resource/Res3>");
+
+		TripleBean t5 = new TripleBean();
+		t5.setS("<http://example.org/resource/Res5>");
+		t5.setP("<http://example.org/property/pro1>");
+		t5.setO("<http://example.org/resource/Res1>");
+
+		TripleBean t6 = new TripleBean();
+		t6.setS("<http://example.org/resource/Res1>");
+		t6.setP("<http://example.org/property/pro1>");
+		t6.setO("<http://example.org/resource/Res:1000>");
+
+		ArrayList<TripleBean> triplesListNoDuplicates = new ArrayList<TripleBean>();
+		triplesListNoDuplicates.add(t1);
+		triplesListNoDuplicates.add(t2);
+		triplesListNoDuplicates.add(t3);
+
+		ArrayList<TripleBean> triplesListWithDuplicates = new ArrayList<TripleBean>();
+		triplesListWithDuplicates.add(t1);
+		triplesListWithDuplicates.add(t2);
+		triplesListWithDuplicates.add(t3);
+		triplesListWithDuplicates.add(t4);
+		triplesListWithDuplicates.add(t5);
+		triplesListWithDuplicates.add(t6);
+
+		// Without duplicates
+		spark().sql("USE triplesWithDuplicates_db");
+		Dataset<Row> expectedTT = spark().createDataset(triplesListNoDuplicates, triplesEncoder).select("s", "p", "o")
+				.orderBy("s", "p", "o");
+		Dataset<Row> actualTT = spark().sql("SELECT s,p,o FROM tripletable ORDER BY s,p,o");
+
+		assertDataFrameEquals(expectedTT, actualTT);
+
+		// Now with duplicates
+		spark().sql("DROP TABLE tripletable");
+		tt_loader = new TripleTableLoader("/triplesWithDuplicates", "triplesWithDuplicates_db", spark(), false, false,
+				false);
+		tt_loader.load();
+
+		expectedTT = spark().createDataset(triplesListWithDuplicates, triplesEncoder).select("s", "p", "o").orderBy("s",
+				"p", "o");
+		actualTT = spark().sql("SELECT s,p,o FROM tripletable ORDER BY s,p,o");
+
+		assertDataFrameEquals(expectedTT, actualTT);
+	}
+
+	/**
+	 * This test verifies an Exception is thrown when an empty file is parsed.
+	 * 
+	 * @throws Exception
+	 */
+	@Test(expected = Exception.class)
+	public void parsingEmptyFile() throws Exception {
+		ClassLoader classLoader = getClass().getClassLoader();
+		File emptyFile = new File(classLoader.getResource("emptyFile.nt").getFile());
+		putFileToHDFS(emptyFile.getAbsolutePath(), "/emptyFile");
+
+		spark().sql("DROP DATABASE IF EXISTS emptyFile_db CASCADE");
+		TripleTableLoader tt_loader = new TripleTableLoader("/emptyFile", "emptyFile_db", spark(), false, false, true);
+		tt_loader.load();
+	}
+
+	// Not behaving as expected. Fix after merging (this belongs to a different
```

Comment:
If it is not working, why do we need to merge it? Just remove the method and add it when it is fixed

Reply:
How is this a reason? We can remove it from the other feature branch.

Reply:
I don't see the problem. This is why the test case is ignored (there is no @Test annotation). It is not introducing errors.